src/index.html

```diff
@@ -12,7 +12,72 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
+    <section class="stars stars--0">
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+    </section>
+    <section class="stars stars--1">
```

Comment:
The `section` tag should be indented with 2 spaces relative to its parent `body` tag.